<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>700</width>
      <height>440</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblCant">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>514</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ebrima</family>
      <pointsize>16</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Cantidad: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="dir">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>621</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ebrima</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Raiz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFolder">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ebrima</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>New Folder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnArchivo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>32</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ebrima</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>New Archivo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBack">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>20</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../../Recursos/File Explorer/back.png</normaloff>../../../../../Recursos/File Explorer/back.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFront">
    <property name="geometry">
     <rect>
      <x>805</x>
      <y>20</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../../Recursos/File Explorer/front.png</normaloff>../../../../../Recursos/File Explorer/front.png</iconset>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>71</y>
      <width>121</width>
      <height>441</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionNuevo_Folder"/>
    <addaction name="actionNuevo_Archivo"/>
    <addaction name="separator"/>
    <addaction name="actionFormatear_Disco"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNuevo_Folder">
   <property name="text">
    <string>Nuevo Folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionNuevo_Archivo">
   <property name="text">
    <string>Nuevo Archivo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionFormatear_Disco">
   <property name="text">
    <string>Formatear Disco</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
